src/NuGet.Core/NuGet.Test.Utility/CommandRunner.cs

```diff
@@ -50,8 +51,8 @@ public class CommandRunner
 
             int exitCode = 1;
 
-            var output = new LockedStringBuilder();
-            var errors = new LockedStringBuilder();
```

Comment:
Yes, I think you can delete the class.  I don't think it's being used anywhere after your changes.  But please confirm.